web/client-api/src/main/java/io/deephaven/web/client/api/tree/JsTreeTable.java

```diff
@@ -307,20 +332,67 @@ public WorkerConnection getConnection() {
         return connection;
     }
 
-    private TicketAndPromise<?> prepareFilter() {
+    private UpdateViewRequest adaptCustomColumn(CustomColumn column) {
+        final String rawString = column.getName() + "=" + column.getExpression();
+        final Selectable columnSpec = new Selectable();
+        columnSpec.setRaw(rawString);
+
+        final UpdateViewRequest request = new UpdateViewRequest();
+        request.setColumnSpec(columnSpec);
+        final int nodeType;
+        if (column.getOptions().rollupNodeType.equals(ROLLUP_NODE_TYPE_AGGREGATED)) {
+            nodeType = RollupNodeType.AGGREGATED.ordinal();
+        } else if (column.getOptions().rollupNodeType.equals(ROLLUP_NODE_TYPE_CONSTITUENT)) {
+            nodeType = RollupNodeType.CONSTITUENT.ordinal();
+        } else {
+            nodeType = RollupNodeType.UNSPECIFIED.ordinal();
```

Comment:
Note that if the rollupNodeType was unspecified (e.g. trees), this would throw. We definitely need tests to cover this.

We also don't need to explicitly set UNSPECIFIED, since it is zero.